index.html

```diff
@@ -476,9 +474,10 @@ <h3>
             When a <a>protocol handler description</a>
             <var>protocol_handler</var> having [=manifest=] <var>manifest</var>
             is invoked, it goes through the same steps used to [=invoke a
-            protocol handler=] defined in [=HTML=], where the user agent SHOULD
-            navigate to [=url=] and the appropriate browsing context is set to
-            a new top level browsing context.
+            protocol handler=] defined in [=HTML=] where the user agent,
```

Comment:
It's usually best not to put which spec stuff if defined in as definitions get moved around to different docs: 

```suggestion
            protocol handler=] where the user agent,
```

Reply:
As a side this launch instruction seems quite iffy to me, I believe it intends to extract the |resultURL| variable in the algorithm referenced to know what URL to navigate to. I think it needs work but it's a bit out of scope for what I'm trying to do here.